src/IntegrationTests/java/uk/gov/hmcts/sscs/tya/CohNotificationsIt.java

```diff
@@ -0,0 +1,140 @@
+package uk.gov.hmcts.sscs.tya;
+
+import static helper.IntegrationTestHelper.*;
+import static java.util.Collections.singletonList;
+import static org.mockito.ArgumentMatchers.argThat;
+import static org.mockito.Mockito.*;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.File;
+import java.io.IOException;
+import javax.servlet.http.HttpServletResponse;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.HttpStatus;
+import org.springframework.mock.web.MockHttpServletResponse;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockHttpServletRequestBuilder;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import uk.gov.hmcts.reform.ccd.client.CoreCaseDataApi;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.sscs.config.NotificationBlacklist;
+import uk.gov.hmcts.sscs.controller.NotificationController;
+import uk.gov.hmcts.sscs.deserialize.CcdResponseWrapperDeserializer;
+import uk.gov.hmcts.sscs.factory.NotificationFactory;
+import uk.gov.hmcts.sscs.service.AuthorisationService;
+import uk.gov.hmcts.sscs.service.NotificationSender;
+import uk.gov.hmcts.sscs.service.NotificationService;
+import uk.gov.hmcts.sscs.service.ReminderService;
+import uk.gov.hmcts.sscs.service.ccd.SearchCcdService;
+import uk.gov.hmcts.sscs.service.coh.CohClient;
+import uk.gov.hmcts.sscs.service.coh.QuestionRound;
+import uk.gov.hmcts.sscs.service.coh.QuestionRounds;
+import uk.gov.hmcts.sscs.service.idam.IdamService;
+import uk.gov.service.notify.NotificationClient;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@ActiveProfiles("integration")
+@AutoConfigureMockMvc
+public class CohNotificationsIt {
```

Comment:
It would be good to follow the pattern we already have around integration/e2e tests. This kind of looks like a mixture of both.